mcp/setup.sh

```diff
@@ -235,6 +240,138 @@ runDeploy() {
   entry
 }
 
+runNewVersion(){
```

Comment:
What about selection of a version which already exists?

Reply:
Added that functionality now